datahub-web-react/src/app/entity/shared/tabs/Dataset/Validations/DatasetAssertionDescription.tsx

```diff
@@ -74,88 +74,90 @@ const getColumnAggregationText = (
             return (
                 <Typography.Text>
-                    Null proportion for column <Typography.Text strong>{field.path}</Typography.Text> is
+                    Null proportion for column <Typography.Text strong>{columnText}</Typography.Text> is
                 </Typography.Text>
             );
         }
         // Numeric Aggregations
         case AssertionStdAggregation.Min: {
             return (
                 <Typography.Text>
-                    Minimum value for column <Typography.Text strong>{field.path}</Typography.Text> is
+                    Minimum value for column <Typography.Text strong>{columnText}</Typography.Text> is
                 </Typography.Text>
             );
         }
         case AssertionStdAggregation.Max: {
             return (
                 <Typography.Text>
-                    Maximum value for column <Typography.Text strong>{field.path}</Typography.Text> is
+                    Maximum value for column <Typography.Text strong>{columnText}</Typography.Text> is
                 </Typography.Text>
             );
         }
         case AssertionStdAggregation.Mean: {
             return (
                 <Typography.Text>
-                    Mean value for column <Typography.Text strong>{field.path}</Typography.Text> is
+                    Mean value for column <Typography.Text strong>{columnText}</Typography.Text> is
                 </Typography.Text>
             );
         }
         case AssertionStdAggregation.Median: {
             return (
                 <Typography.Text>
-                    Median value for column <Typography.Text strong>{field.path}</Typography.Text> is
+                    Median value for column <Typography.Text strong>{columnText}</Typography.Text> is
                 </Typography.Text>
             );
         }
         case AssertionStdAggregation.Stddev: {
             return (
                 <Typography.Text>
-                    Standard deviation for column <Typography.Text strong>{field.path}</Typography.Text> is
+                    Standard deviation for column <Typography.Text strong>{columnText}</Typography.Text> is
                 </Typography.Text>
             );
         }
         // Native Aggregations
         case AssertionStdAggregation.Native: {
             return (
                 <Typography.Text>
-                    Column <Typography.Text strong>{field.path}</Typography.Text> values are
+                    Column <Typography.Text strong>{columnText}</Typography.Text> values are
                 </Typography.Text>
             );
         }
         default:
             // No aggregation on the column at hand. Treat the column as a set of values.
             return (
                 <Typography.Text>
-                    Column <Typography.Text strong>{field.path}</Typography.Text> values are
+                    Column <Typography.Text strong>{columnText}</Typography.Text> values are
```

Comment:
Do we need to handle edge case on line 180 also?

Reply:
Or line 65

Reply:
Or line 48

Reply:
oh no I don't believe so. If fields is undefined, we pass in `undefined` same thing if `fields` is an empty list (the situation we were seeing with some users)

Reply:
oh wait I didn't see your other two comments (github was stale)

Reply:
but yes going to handle those other edge cases where we throw